acceso/views.py

```diff
@@ -78,7 +78,7 @@ def main(request, options):
     return render(request, 'acceso/index.html', context)
 
 def filtrar_imagenes(request):
-    filter_list = ["", "none"] #Don't know how to add filters since images models are being created dynamically
+    filter_list = ["test1", "apple"] #Don't know how to add filters since images models are being created dynamically
```

Comment:
We'll want to change this once there are filters in the database, so that the filter buttons are created dynamically based on the values in the db.